models/classes/runner/QtiRunnerService.php

```diff
@@ -1503,7 +1503,7 @@ protected function onTimeout(RunnerServiceContext $context, AssessmentTestSessio
         switch ($timeOutException->getCode()) {
             case AssessmentTestSessionException::ASSESSMENT_TEST_DURATION_OVERFLOW:
                 \common_Logger::i('TIMEOUT: closing the assessment test session');
-                $session->endTestSession();
+                $session->moveThroughAndEndTestSession();
```

Comment:
Considering, this is a method, appearing in a very specific version of QTI-SDK, I strongly recommend explicitly requiring that version in `composer.json` as `^0.25` once it's been released.

Reply:
Yep, good suggestion, I will do that when the tag will be released in `qti-sdk`